platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -342,6 +342,8 @@ protected void onGodotSetupCompleted() {
 			if (inputHandler != null) {
 				inputHandler.enableLongPress(longPressEnabled);
 				inputHandler.enablePanningAndScalingGestures(panScaleEnabled);
+				int rotaryInputAxis = java.lang.Integer.parseInt(GodotLib.getGlobal("input_devices/pointing/android/rotary_input_scroll_axis"));
```

Comment:
This logic should be outside of the `runOnUiThread` block (and below `boolean panScaleEnabled = ...`) because the settings are being retrieved on the render thread.
Only the line below should remain within this block.

Reply:
fixed